.pre-commit-config.yaml

```diff
@@ -30,3 +30,9 @@ repos:
     rev: 5.13.2
     hooks:
       - id: isort
+
+  - repo: https://github.com/adamchainz/django-upgrade
+    rev: 1.15.0
+    hooks:
+      - id: django-upgrade
+        args: [--target-version, "5.0"] # Replace with Django version
```

Comment:
This is not clear from the documentation of that tool.  It is also unclear if I target `5.0`, will all previous upgrades also be run?

I have had difficulty upgrading from one version to the next without django-upgrading to that new version.

Reply:
Do any of the changes made below break Django 3.2 or 4.2?

Reply:
> This is not clear from the documentation of that tool.

I don't think this should be expected from that tool. It's more about where and why it's being used. This could make perfect sense if there were only supported version, or we were 5.0 as the minimum supported version.

> It is also unclear if I target 5.0, will all previous upgrades also be run?

I think this is actually clear:

> Pass your Django version as <major>.<minor> to the --target-version flag. django-upgrade will run all its fixers for versions up to and including the target version. 

https://github.com/adamchainz/django-upgrade?tab=readme-ov-file#usage

Reply:
The setup was nice except for the target version. I think it should be in the repo, but branch name is losing it's connection with the PR context by every review comments 🙁